misc/dist/linux/macros.godot

```diff
@@ -0,0 +1,41 @@
+#
+# RPM macros for godot
+#
+%__godot %{_bindir}/godot-headless
```

Comment:
This probably needs to be filled in by the Godot build system.

Reply:
Are you referring to the %{_bindir}? If so, how would you suggest that being filled in?

Reply:
No, I don't want the buildsystem to have anything to do with the distribution-specific files. We currently don't provide any `scons install` feature either, it's all manual.